src/styles/_variables.scss

```diff
@@ -0,0 +1,120 @@
+// Typography
+// ------------------------------------------------
+
+$font__default: "Satoshi Variable", sans-serif;
+$font__code: "PT Mono", monospace;
+
+// Colors
+// ------------------------------------------------
+
+// Brand colors
+$color__c4-button-blurple: rgba(113, 74, 235, 1);
+$color__c4-logo-blurple: #855eff;
+$color__sponsor-color: rgba(64, 203, 161, 1);
+
+// Black and white
+$color__white: rgba(255, 255, 255, 1);
+$color__black: rgba(2, 2, 3, 1);
+
+// Neutral palette
+$color__n-100: rgba(11, 10, 13, 1);
+$color__n-90: rgba(29, 27, 38, 1);
+$color__n-80: rgba(46, 43, 59, 1);
+$color__n-70: rgba(49, 46, 64, 1);
+$color__n-60: rgba(64, 61, 77, 1);
+$color__n-50: rgba(75, 71, 89, 1);
+$color__n-40: rgba(86, 82, 102, 1);
+$color__n-30: rgba(107, 102, 128, 1);
+$color__n-20: rgba(141, 138, 153, 1);
+$color__n-10: rgba(173, 171, 178, 1);
+$color__n-5: rgba(196, 194, 204, 1);
+$color__n-2: rgba(230, 230, 230, 1);
+
+// Spacing
+// ------------------------------------------------
+
+$spacing__base: 0.5rem; // 8px
+
+$spacing__xxs: calc(#{$spacing__base}/ 4); // 2px
+$spacing__xs: calc(#{$spacing__base}/ 2); // 4px
+$spacing__s: $spacing__base; // 8px
+$spacing__m: calc(#{$spacing__base} * 2); // 16px
+$spacing__l: calc(#{$spacing__base} * 3); // 24px
+$spacing__xl: calc(#{$spacing__base} * 4); // 32px
+$spacing__xxl: calc(#{$spacing__base} * 5); // 40px
+$spacing__xxxl: calc(#{$spacing__base} * 6); // 48px
+
+// Breakpoints
+// ------------------------------------------------
+
+$breakpoint__xs: 0;
+$breakpoint__s: 600px;
+$breakpoint__m: 960px;
+$breakpoint__l: 1280px;
+$breakpoint__xl: 1920px;
+
+// Layout
+// ------------------------------------------------
+
+$layout__max-width: 1280px;
+
+// Shadows
+// ------------------------------------------------
+
+$shadow__base: 0 0 0.5rem 0 rgba(0, 0, 0, 0.5);
+$shadow__hover: 0 0 0.5rem 0 rgba(0, 0, 0, 0.75);
+
+// Transitions
+// ------------------------------------------------
+
+$transition__base: 0.2s ease-in-out;
+
+// Border radius
+// ------------------------------------------------
+
+$border-radius__base: 0.2rem; // 4px
```

Comment:
`$spacing__xs`?

Reply:
Decided to keep this one as its own thing so we can adjust it separately